kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -341,11 +341,16 @@ define([
                         $(e.currentTarget).off('click');
                         this.updatePathFn(this.path += '/' + $(e.currentTarget).data().name);
                     });
+
                     $('td:eq(4)', nRow).find('select').select2({
-                        placeholder: 'Select format'
+                        placeholder: 'Select a type',
+                        containerCssClass: 'kb-data-staging-import-dropdown'
```

Comment:
Checking out the DOM that `select2` generates, what it considers the 'container' is not what I'd call the container! :\  I don't know, having looked at the html / css, I think it might be cleaner to go back to what you had before, but just add an extra `<span class="kb-data-staging-import__select2_container">` around the `<select>` element, so then at least you've separated out the container div with the negative left margin, and you have all the select2 stuff within that specific container. Urgh!

Reply:
i know i completely agree that isn't the ideal container to be playing off of. the good news is i have a fix for the need to style that div with a negative left margin (to come in next PR). i'm also refactoring the class names a bit overall in this table so hopefully it will come out a bit cleaner in the end 🤞 still doing some tweaking but once i get a bit closer maybe we can pair on it to discuss